include/irr/asset/IImageAssetHandlerBase.h

```diff
@@ -31,10 +37,207 @@ class IImageAssetHandlerBase : public virtual core::IReferenceCounted
 			return width;
 		}
 
+		static inline core::vector3du32_SIMD calcPitchInBlocks(uint32_t width, uint32_t height, uint32_t depth, uint32_t blockByteSize)
+		{
+			constexpr auto VALUE_FOR_ALIGNMENT = 1;
+			core::vector3du32_SIMD retVal;
+			retVal.X = calcPitchInBlocks(width, blockByteSize);
+			retVal.Y = calcPitchInBlocks(height, VALUE_FOR_ALIGNMENT);
+			retVal.Z = calcPitchInBlocks(depth, VALUE_FOR_ALIGNMENT);
```

Comment:
this will fuck up, you should probably use `blockByteSize*width` for the Y block byte size, and then `blockByteSize*width*height` for Z

Reply:
"fixed" in upcoming commit